app/views/facilities/_facilities_list.html.haml

```diff
@@ -4,7 +4,7 @@
     - @azlist.map.with_index do |(letter, matched_facilities), index|
       - matched_facilities.each do |facility|
         %li.facility_listing.js--facility_listing{ class: [ az_classname_for_facility(index, letter), @list_layout ] }
-          - if SettingsHelper.feature_on?(:facility_tile_list)
+          - if SettingsHelper.feature_on?(:facility_tile_list) && SettingsHelper.feature_on?(:facility_tile_list_admin)
```

Comment:
I think I would lean towards leaving this as is - if someone wanted to use the tile view on the front end with the default image, that's fine with me